lib/ex_sdp/media.ex

```diff
@@ -77,6 +77,12 @@ defmodule ExSDP.Media do
   @spec get_attributes(media :: t(), key :: module() | atom() | binary()) :: [Attribute.t()]
   def get_attributes(media, key), do: Utils.get_attributes(media, key)
 
+  @spec delete_attribute(media :: t(), key :: module() | atom() | binary()) :: t()
+  def delete_attribute(media, key), do: Utils.delete_attribute(media, key)
+
+  @spec delete_attributes(media :: t(), [key :: module() | atom() | binary()]) :: t()
+  def delete_attributes(media, keys), do: Utils.delete_attributes(media, keys)
```

Comment:
I wonder if we should instead put everything in ExSDP and instead of distinguisng between ExSDP.add/get/delete_attribute and ExSDP.Media.add/get/delete_attribute, always have ExSDP.add/get/delete_attribute :thinking:

Reply:
imo yes, in general the distinction between `ExSDP.get_attribute()` and `ExSDP.Media.get_attribute()` is weird and prone to mistakes.